turbomoleio/core/molecule.py

```diff
@@ -69,7 +97,69 @@ def get_mol_and_indices_frozen(string):
         if len(s) == 5 and s[-1].lower() == "f":
             frozen_indices.add(iatom)
 
-    mol = Molecule(species, coords)
+    coords = np.array(coords)
+    if cell_string is not None:
+        periodicity = int(periodic_string.strip())
+        cell = [float(sp) for sp in cell_string.strip().split()]
+        if periodicity == 1:
+            if len(cell) != 1:
+                raise ValueError('The $cell data group should contain 1 number for 1D periodic systems')
+            yrange = np.max(coords[:, 1])-np.min(coords[:, 1])
+            zrange = np.max(coords[:, 2])-np.min(coords[:, 2])
+            lattice = Lattice.orthorhombic(a=bohr_to_ang*cell[0],
+                                           b=bohr_to_ang*yrange+periodic_extension,
+                                           c=bohr_to_ang*zrange+periodic_extension)
+        elif periodicity == 2:
+            if len(cell) != 3:
+                raise ValueError('The $cell data group should contain 3 numbers for 2D periodic systems')
+            zrange = np.max(coords[:, 2])-np.min(coords[:, 2])
+            lattice = Lattice.from_parameters(a=bohr_to_ang*cell[0],
```

Comment:
isn't the `vesta=True` option needed here and below?

Reply:
Indeed. Done in base.py